chart/templates/postgres.yaml

```diff
@@ -0,0 +1,80 @@
+{{- if eq .Values.db.enabled true }}
```

Comment:
We would need a helm post-install hook to create the SQL db's

Reply:
@moshloop, what do you think of [this](https://stackoverflow.com/a/26599273) approach to create DBs instead of creating a post-install job?